app/src/marketplace/components/GoogleAnalyticsTracker/index.jsx

```diff
@@ -1,33 +0,0 @@
-// @flow
-
-import { useEffect } from 'react'
-import { withRouter, type Location } from 'react-router-dom'
-import ReactGA from 'react-ga'
-
-const gaId = process.env.GOOGLE_ANALYTICS_ID
-
-type Props = {
-    location: Location,
-}
-
-export default withRouter(({ location = {} }: Props) => {
-    if (!gaId) {
-        return null
```

Comment:
See that conditionally triggered `useEffect` below? Not ideal for react. I fixed it.